docs/source/docker/getting_started/first_steps.rst

```diff
@@ -362,6 +362,43 @@ you might want to change:
     )
 
 
+**Checkpoints** from your training process will be stored in the output directory.
```

Comment:
Let's add an example here: 

Just a suggestion:

The output directory is defined in the docker run command as shown in #volume-mapping. 
If your docker run command looks like this:
```console
docker run --gpus all --rm -it \
    -v /home/mymachine/lightly-output/:/home/output_dir \
    lightly/worker:latest \
    token=MY_AWESOME_TOKEN \
    worker.worker_id=MY_WORKER_ID
```
The checkpoint path would be something like this: `/home/mymachine/lightly-output/my_checkpoint_file.ckpt`